source/_integrations/modbus.markdown

```diff
@@ -220,11 +230,60 @@ slave:
   default: 0
 {% endconfiguration %}
 
-### Configuring platform binary sensor
+### Configuring data_type and struct
+
+Climate and Sensor share setup of data_type and struct. 
+
+```yaml
+# Example configuration.yaml entry for platform common parameters
+modbus:
+  - type: tcp
+    host: IP_ADDRESS_1
+    port: 2020
+    name: "hub1"
+    sensors:
+      - name: sensor1
+        data_type: int
+```
+
+{% configuration %}
+data_type:
+    description: Response representation (int16, int32, int64, uint16, uint32, uint64, float16, float32, float64, string). `int/uint`are silently converted to `int16/uint16`.
+    required: false
+    type: string
+    default: int16
+offset:
+      description: Final offset (output = scale * value + offset).
+      required: false
+      type: float
+      default: 0
+precision:
+      description: Number of valid decimals.
+      required: false
+      type: integer
+      default: 1
+scale:
+      description: Scale factor (output = scale * value + offset).
+      required: false
+      type: float
+      default: 1
+structure:
+      description: "If `data_type` is custom specified a double-quoted Python struct is expected here, to format the string to unpack the value. See Python documentation for details. Example: `>i`."
+      required: false
+      type: string
+      default: ">f"
+swap:
+      description: "Swap the order of bytes/words, options are `none`, `byte`, `word`, `word_byte`."
+      required: false
+      default: none
+      type: string
```

Comment:
I see that you use here 4/5 indents but it should be 2 😉

Reply:
done